pandas/sparse/scipy_sparse.py

```
@@ -0,0 +1,102 @@
+    # index and column levels must be a partition of the index
+    _check_is_partition([ilevels, jlevels], range(ss.index.nlevels))
+
+    # from the SparseSeries: get the labels and data for non-null entries
+    values = ss._data.values._valid_sp_values
+    blocs = ss._data.values.sp_index.blocs
+    blength = ss._data.values.sp_index.blengths
+    nonnull_labels = list(
+        itertools.chain(*[ss.index.values[i:(i + j)] for i, j in zip(blocs, blength)]))
+
+    def get_indexers(levels):
+        """ Return sparse coords and dense labels for subset levels """
+        values_ilabels = [tuple(x[i] for i in levels) for x in nonnull_labels]
+        labels_to_i = _get_index_subset_to_coord_dict(
+            ss.index, levels, sort_labels=sort_labels)
+        i_coord = [labels_to_i[i] for i in values_ilabels]
+        return(i_coord, list(labels_to_i.keys()))
+
+    i_coord, i_labels = get_indexers(ilevels)
+    j_coord, j_labels = get_indexers(jlevels)
+
+    return(values, i_coord, j_coord, i_labels, j_labels)
+
+
+def _sparse_series_to_coo(ss, ilevels=(0,), jlevels=(1,), sort_labels=False):
+    """ Convert a SparseSeries to a scipy.sparse.coo_matrix using index levels ilevels, jlevels as the row and column
+    labels respectively. Returns the sparse_matrix, row and column labels. """
+    if ss.index.nlevels < 2:
+        raise ValueError('to_coo requires MultiIndex with nlevels > 2')
+    if not ss.index.is_unique:
+        raise ValueError(
+            'Duplicate index entries are not allowed in to_coo transformation.')
+
+    # to keep things simple, only rely on integer indexing (not labels)
+    ilevels = [ss.index._get_level_number(x) for x in ilevels]
+    jlevels = [ss.index._get_level_number(x) for x in jlevels]
+    ss = ss.copy()
+    ss.index.names = [None] * ss.index.nlevels  # kill any existing labels
+
+    v, i, j, il, jl = _to_ijv(
+        ss, ilevels=ilevels, jlevels=jlevels, sort_labels=sort_labels)
+    import scipy.sparse
```

Better to import scipy.sparse at the top of the function so things fail faster if the import isn't available -- it's also slightly more standard, though I can't find it in PEP8